typescript/src/Writer.ts

```diff
@@ -0,0 +1,77 @@
+export const LITTLE_ENDIAN = true;
```

Comment:
? I don't understand why we export this.

Reply:
Probably happened automatically when I did a "move to a separate file" refactor